docs/source/changelog.rst

```diff
@@ -21,6 +21,72 @@ Changelog
 
 .. _latest:
 
+.. _`v0-13-0`:
+
+0.13.0 / 2023-11-09
+###################
+
+.. image:: https://zenodo.org/badge/DOI/10.5281/zenodo.8252548.svg
```

Comment:
I guess we can live with a bad DOI for a short while and fix this after the release?

Reply:
I think that's what we usually do? But if we wait for the new DOI we can update then merge.

Reply:
In the past I've created the entry without badge, merged, and then added the badge after the release with another PR. We had some "almost finished" releases that took a week to get finished where I was a bit premature with adding a release date, so I try to have a consistent state online. Merging the changelog after release is also possible. It will not be included properly in the repo dump in that case, but I don't think anyone cares! :-D